src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -472,18 +472,20 @@ public function multi_resize( $sizes ) {
 	 * Create an image sub-size and return the image meta data value for it.
 	 *
 	 * @since 5.3.0
+	 * @since 6.0.0 Added the $mime_type parameter.
 	 *
-	 * @param array $size_data {
+	 * @param array  $size_data {
 	 *     Array of size data.
 	 *
 	 *     @type int  $width  The maximum width in pixels.
 	 *     @type int  $height The maximum height in pixels.
 	 *     @type bool $crop   Whether to crop the image to exact dimensions.
 	 * }
+	 * @param string $mime_type Optional. The mime type of the image.
 	 * @return array|WP_Error The image data array for inclusion in the `sizes` array in the image meta,
 	 *                        WP_Error object on error.
 	 */
-	public function make_subsize( $size_data ) {
+	public function make_subsize( $size_data, $mime_type = null ) {
```

Comment:
See above, this needs to be called out in a dev note.

Reply:
👍🏼  will note on trac ticket